src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -57,7 +55,7 @@ public class EditCommand extends Command {
         "This person already exists in the contact list. Please use a different name.";
     public static final String MESSAGE_EDIT_ROLE_CONTAIN_AFFILIATION =
             "This person contains affiliations. Changing of Role is not allowed.";
-
+    public static final String MESSAGE_EDIT_ROLE_NOT_ALLOW = "Edit of role is not allow.";
```

Comment:
Dear sir, grammar sir

Reply:
Opps... man's england not powderful enough hahaha, will change it in the next commit.

Reply:
thanks you sir